Cargo.lock

```diff
@@ -4315,7 +4327,17 @@ version = "0.15.0"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "e9c9dc66cc29792b663ffb5269be669f1613664e69ad56441fdb895c2347b930"
 dependencies = [
- "rstest_macros",
+ "rstest_macros 0.14.0",
+ "rustc_version 0.4.0",
+]
+
+[[package]]
+name = "rstest"
+version = "0.16.0"
```

Comment:
It looks like we're building 2 versions of rstest now, because it was updated for `nu-cli` but not other crates.

I would vote to leave these unrelated version bumps out of the PR, but it's not a hill I'll die on.

Reply:
I've unbumped these for you.

Reply:
ok.